src/Poseidon/Package.hs

```diff
@@ -576,6 +578,23 @@ newMinimalPackageTemplate baseDir name (GenotypeDataSpec format_ geno _ snp _ in
     ,   posPacDuplicate = 1
     }
 
+makePseudoPackageFromInGenotypeData :: InGenotypeData -> IO PoseidonPackage
+makePseudoPackageFromInGenotypeData (InGenotypeData format_ genoFile_ snpFile_ indFile_ snpSet_) = do
+    let genotypeData = GenotypeDataSpec format_ genoFile_ Nothing snpFile_ Nothing indFile_ Nothing (Just snpSet_)
+        baseDir = getBaseDir genoFile_ snpFile_ indFile_
+        pacName = takeBaseName genoFile_
+    inds <- loadIndividuals "." genotypeData
+    newPackageTemplate baseDir pacName genotypeData (Just (Left inds)) []
+    where
+        getBaseDir :: FilePath -> FilePath -> FilePath -> FilePath
+        getBaseDir g s i =
+            let baseDirGeno = takeDirectory genoFile_
+                baseDirSnp = takeDirectory snpFile_
+                baseDirInd = takeDirectory indFile_
+            in if baseDirGeno == baseDirSnp && baseDirSnp == baseDirInd
+               then baseDirGeno
+               else throwM $ PoseidonUnequalBaseDirException g s i
```

Comment:
I think this is actually an unnecessary restriction. Of course it will rarely matter, since most users would indeed keep the three files in one directory, but we really don't need this requirement at all. The `baseDir` is just a root directory   off of which the individual file names are relative. So instead of searching for a common base-directory of these three files, we could simply take the current working directory as the root directory, right? The user specifies the three files in either absolute or relative paths, but if they are relative, they are meant to be relative from the current working directory, so we can just take that as the baseDir. We _should_ however check that the directories given by the user aren't absolute, in which case we'd probably have to take care that either we set `/` as the baseDir, or we check that our path logic still works with absolute paths if we stick with the CWD as the baseDir.

Am I making sense?

Reply:
(my point is: since we never will write out that package, the baseDir here is just a search path...)

Reply:
Ok - that was maybe unnecessarily restrictive. c57e2b83df52c663c666fadebcd2a82ad11bcdd3 replaces it with a system that simply takes all relative paths

Reply:
So to explain this a bit more: For genoconvert we need a clear basedir to write to, because where else would you write? `.` doesn't make sense.

Maybe we should add an explicit output directory option for genoconvert, but I want to keep the default to write where the other data already is. That's just convenient. A user-specified output dir would have to be provided for every input package/pseudopackage. A lot of typing for little gain, so even if we add this, it should be optional.

Another option would be to write to the location of one or multiple of the input files. But that's also not very elegant and might become confusing easily.

I would argue, sometimes restrictions make a program more easy to use.

Reply:
Hmm, not following 100%. There is a clear expectation from the user that the new genotype files will be written exactly to where the old ones reside. So the idea here is to take whatever the input paths are and simply change the extension from `.bed` to `.geno` and from `.bim` to `.snp` and so on. That should then preserve the relative paths given to the programs. In 99,9% of cases, users will simply have the three files in one place, so this will all be the same to your restriction. In the rare case where some users might have the three files spread out over several directories, our program would simply follow that and place the new three files alongside the old ones. 

I see no problem.

Reply:
I considered this, but not all genotype data formats consist of these three files. This will fall apart if/when we come back to #150

Reply:
OK fine then :-) We're discussing a marginal thing, I admit. I think most people won't even feel this "restriction". So fine to go ahead with the restriction.